src/domain/event/eventCategorisation/EventCategorisation.tsx

```diff
@@ -21,6 +21,20 @@ const EventCategorisation: React.FC<Props> = ({ eventData, language }) => {
   const inLanguage = eventData.event?.inLanguage;
   const audience = eventData.event?.audience;
   const keywords = eventData.event?.keywords;
+  const offer = eventData.event?.offers?.[0];
+  const price = offer?.price?.[language] || '';
+  const priceDescription = offer?.description?.[language] || '';
+  const isFree = offer?.isFree ?? !price;
+  const priceInfoUrl = offer?.infoUrl?.[language] || '';
```

Comment:
Like in the other PR this can be cleaned a tiny bit:

```
  const offer = eventData.event?.offers?.[0];
  const price = offer?.price?.[language];
  const priceDescription = offer?.description?.[language];
  const isFree = offer?.isFree ?? !price;
  const priceInfoUrl = offer?.infoUrl?.[language];
```

just a small nitpick

Reply:
(default values are not needed)

Reply:
Default values removed